drop-in/src/main/java/com/adyen/checkout/dropin/DropIn.kt

```diff
@@ -41,32 +44,63 @@ class DropIn private constructor() {
          * Starts the checkout flow to be handled by the Drop-In solution. Make sure you have [DropInService] set up before calling this.
          * We suggest that you set up the resultHandlerIntent with the appropriate flags to clear the stack of the checkout activities.
```

Comment:
I think we can explain the result options here. Something like:

"Starts the checkout flow to be handled by the Drop-In. Make sure you have [DropInService] set up before calling this.

You will receive the the result of the DropIn in `onActivityResult`, either payment finished, canceled or error.
You can use `RESULT_KEY` and `RESULT_ERROR_REASON` to get the contents.

Anternatively, you can specify a `resultHandlerIntent` in the `DropInConfiguration` pointing to one of your activities, which will be called in case the payment finishes successfully. We suggest that you set up the resultHandlerIntent with the appropriate flags to clear the stack of the checkout activities."